go-controller/pkg/ovn/base_network_controller_egressip.go

```diff
@@ -1644,7 +1674,7 @@ func (e *egressIPZoneController) addPodEgressIPAssignment(egressIPName string, s
 	// exec when node is local OR when pods are local
 	// don't add a reroute policy if the egress node towards which we are adding this doesn't exist
 	if loadedEgressNode && loadedPodNode && isLocalZonePod {
-		ops, err = e.createReroutePolicyOps(ops, podIPs, status, egressIPName, nextHopIP)
+		ops, err = e.createReroutePolicyOps(ops, podIPs, status, mark, egressIPName, nextHopIP)
```

Comment:
isn't this redunant to do pkt marking here ? this is already done in GW router, also mark doesn't survive when it goes out to wire.

Reply:
I was thinking of EIP multi NIC when I wrote this where it can reroute to local mgnt port.